### PR TITLE
Add appendChildren() and removeChildren() function to XML

### PR DESCRIPTION
**Pull request by SupunS**
Fix https://github.com/ballerina-platform/ballerina-lang/issues/9816

### CODE REVIEW

File: bvm/ballerina-core/src/main/java/org/ballerinalang/model/values/BXMLSequence.java [hunk 1]
```diff
@@ -462,4 +462,13 @@ public boolean hasNext() {
             return cursor < value.sequence.size();
         }
     }
+
+    @Override
+    public void removeChildren(String qname) {
+        if (sequence.size() != 1) {
```
Comment (Kishanthan) [inline bvm/ballerina-core/src/main/java/org/ballerinalang/model/values/BXMLSequence.java]: Shouldn't this check be < 1 instead? Also what if the size is zero?

Comment (SupunS) [reply bvm/ballerina-core/src/main/java/org/ballerinalang/model/values/BXMLSequence.java]: size != 1 will work.

size > 1 means its a sequence, which means its not an element, hence cannot remove a children.
size is zero means its an empty sequence. In that case again we can't remove..

File: tests/ballerina-unit-test/src/test/resources/test-src/types/xml/xml-native-functions.bal [hunk 2]
```diff
@@ -842,3 +842,45 @@ function testGetChildrenOfSequence() returns (int, xml) {
...
+    return (children, x1.*);
+}
+
+function testRemoveChildrenWithNamesapces() returns (xml, xml) {
+    xmlns "http://wso2.com" as ns0;
+    xml x1 = xml `<person><name>John</name><ns0:name>Foo</ns0:name><age>50</age><name>Doe</name></person>`;
+    xml children = x1.*;
+    x1.removeChildren(ns0:name);
+    return (children, x1.*);
+}
```
Comment (hasithaa) [inline tests/ballerina-unit-test/src/test/resources/test-src/types/xml/xml-native-functions.bal]: Shall we add a test case to verify multiple level of xml removing?

Comment (SupunS) [reply tests/ballerina-unit-test/src/test/resources/test-src/types/xml/xml-native-functions.bal]: added

**Pull request merged**